modules/ossm-installation-activities.adoc

```diff
@@ -8,7 +8,7 @@
 [id="ossm-installation-activities_{context}"]
 = Operator overview
 
-{SMProductName} requires the following four Operators:
+{SMProductName} requires the following Operators:
```

Comment:
I'd delete "following." It doesn't really add anything.

Reply:
I believe we should keep "following". Just below this section we have, [Installing the Operators](https://docs.openshift.com/rosa/service_mesh/v2x/installing-ossm.html#ossm-install-ossm-operator_installing-ossm) section there also we can see the sentence, "To install Red Hat OpenShift Service Mesh, install the _following_ Operators in this order."

Please let me know your thoughts. Thanks!

Reply:
Hello @nalhadef . I believe we should keep "following". Just below this section we have, [Installing the Operators](https://docs.openshift.com/rosa/service_mesh/v2x/installing-ossm.html#ossm-install-ossm-operator_installing-ossm) section there also we can see the sentence, "To install Red Hat OpenShift Service Mesh, install the _following_ Operators in this order."

Please let me know your thoughts. Thanks!